doc/mgr/modules.rst

```diff
@@ -48,13 +48,58 @@ or older versions of Ceph.
 Logging
 -------
 
-``MgrModule`` instances have a ``log`` property which is a logger instance that
-sends log messages into the Ceph logging layer where they will be recorded
-in the mgr daemon's log file.
+Logging in Ceph manager modules is done as in any other Python program. Just
+import the ``logging`` package and get a logger instance with the
+``logging.getLogger`` function.
+
+Each module has a ``log_level`` option that specifies the current Python
+logging level of the module.
+To change or query the logging level of the module use the following Ceph
+commands::
+
+  ceph config get mgr mgr/<module_name>/log_level
+  ceph config set mgr mgr/<module_name>/log_level <INFO|DEBUG|CRITICAL|ERROR|WARNING|>
```

Comment:
Should be `<info|debug|critical|error|warning>` instead of `<INFO|DEBUG|CRITICAL|ERROR|WARNING|>`.